packages/docs-app/src/examples/core-examples/tabsExample.tsx

```diff
@@ -66,24 +94,32 @@ export class TabsExample extends React.PureComponent<IExampleProps, ITabsExample
                     </Navbar.Group>
                 </Navbar>
                 {/* uncontrolled mode & each Tab has a panel: */}
-                <Tabs
-                    animate={this.state.animate}
-                    id="TabsExample"
-                    key={this.state.vertical ? "vertical" : "horizontal"}
-                    renderActiveTabPanelOnly={this.state.activePanelOnly}
-                    vertical={this.state.vertical}
-                >
-                    <Tab id="rx" title="React" panel={<ReactPanel />} />
-                    <Tab id="ng" title="Angular" panel={<AngularPanel />} />
-                    <Tab id="mb" title="Ember" panel={<EmberPanel />} />
-                    <Tab id="bb" disabled={true} title="Backbone" panel={<BackbonePanel />} />
-                    <Tabs.Expander />
-                    <InputGroup className={Classes.FILL} type="text" placeholder="Search..." />
-                </Tabs>
+                <div style={{ width: `${this.state.overflowWidth}%`, marginRight: "auto" }}>
```

Comment:
this makes for some awful panel visuals. i would make only the top example (the `Navbar`) overflow because it has no panels and leave this one untouched.
![image](https://user-images.githubusercontent.com/464822/44555809-d112dd80-a6eb-11e8-9d5b-58950502482c.png)

Reply:
It does technically need some sort of panel so the didChildrenChange in the componeneDidUpdate can figure out the panel width changed. So if you don't have a panel of any sort the animation of the indicator would not work when you resize the component. Though we can add something to make it work without the panels, would just need to cache the previous widths somewhere to compare